rules/msk_topic_name.md

```diff
@@ -0,0 +1,53 @@
+# msk_topic_name
+
+## Requirements
+
+An MSK topic must have the name prefixed with the team name or one of the configured aliases for that team.
+
+## Configuration
+
+```hcl
+rule "msk_topic_name" {
+  enabled = true
+  team_aliases = {
+    pubsub = ["alias_pubsub1", "alias_pubsub2"]
+    iam = ["auth", "auth-customer"]
+  }
+}
+```
+
+`team_aliases` maps a team name to it's allowed aliases.
+
+## Example
+
+### Good example
+
+Good for team `pubsub` :
+```hcl
+resource "kafka_topic" "good_topic1" {
+	name = "pubsub.good-topic"
```

Comment:
pedantic nit: this example uses tabs for indenting, the others use spaces

Reply:
Fixed in https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/11/commits/c3117755bfe78eec16ee09b83bf094ab5e4de6dd